data/phi1020/phi001/__cts__.xml

```diff
@@ -1,10 +1,10 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi1020" projid="latinLit:phi001" urn="urn:cts:latinLit:phi1020.phi001" xml:lang="lat">
-            <ti:title xml:lang="eng">Thebais</ti:title>
+            <ti:title xml:lang="lat">Thebais</ti:title>
             <ti:edition projid="latinLit:perseus-lat2" urn="urn:cts:latinLit:phi1020.phi001.perseus-lat2" workUrn="urn:cts:latinLit:phi1020.phi001">
-                <ti:label xml:lang="eng">Thebais</ti:label>
-                <ti:description xml:lang="eng">Statius, P. Papinius. Statius, Vol I-II. John Henry
-                    Mozley. London: William Heinemann; New York: G.P. Putnam&amp;apos;s Sons. 1928.
-                    OCR.</ti:description>
+                <ti:label xml:lang="lat">Thebais</ti:label>
+                <ti:description xml:lang="mul">Statius, P. Papinius. Statius, Volumes 1-2. 
+                    Mozley, John Henry, editor. London: William Heinemann; New York: G.P. Putnam's Sons, 1928.
```

Comment:
You can keep the entity here — it's the only way to get a curly quote. Doesn't matter here.
Same with an ampersand.

Reply:
The reason I eliminated the ampersand is that there isn't any ampersand in the way the name is printed on the volume. I eliminated the apos; one because in the Scaife Viewer it appears as follows:

G.P. Putnam&apos;s Sons.  Was the entity character incorrect in the files?

Reply:
`&apos;` is the entity for the curly apostrophe. What you changed it to is the straight quotation mark.

Reply:
Oh, I see the source. They converted the ampersand in the entity to another entity. sigh. It may be that the Scaife just doesn't interpret the entities correctly. — (it should.)  I have replaced `&` with `and` for this reason. I think that curly quotes are ok, though. `’` vs `'`